nms/app/views/organizations/OrganizationEdit.js

```diff
@@ -160,24 +159,50 @@ function OrganizationEdit(props: WithAlert & Props) {
   const [organization, setOrganization] = useState<?Organization>(null);
   const tableRef = React.createRef();
   const [organizationToDelete, setOrganizationToDelete] = useState(null);
-  const orgRequest = useAxios<null, {organization: Organization}>({
-    method: 'get',
-    url: '/host/organization/async/' + params.name,
-    onResponse: useCallback(res => {
-      setOrganization(res.data.organization);
-    }, []),
-  });
+  const [_allNetworks, setAllNetworks] = useState(null);
+  // const [organizations, setOrganizations] = useState(null);
+  // const enqueueSnack/bar = useEnqueueSnackbar();
+  const [isAllNetworksLoading, setIsAllNetworksLoading] = useState(true);
+  const [isOrganizationLoading, setIsOrganizationLoading] = useState(true);
 
-  const networksRequest = useAxios({
```

Comment:
Also here I don't understand why we not using `useAxios`? Was this causing issue with mocking in the tests? I'm sure this can be addressed.